### DISCUSSION TITLE
使用 shadowrockets 配合规则来访问 UnblockNeteaseMusic 的 server, 但是不生效, 是不能这么用吗

### DISCUSSION THREAD

**Discussion by heeroyuyj** [1715173329/UnblockNeteaseMusic#216] [Q&A] [answered]
测试效果是手机上看无用, 灰色歌曲还是提示无版权, 电脑端 cmd 内也没有相关连接访问的提示出现

**Comment by zq0x7a0x71** [comment]
> 测试效果是手机上看无用，灰色歌曲还是提示无版权，电脑端 cmd 内也没有相关连接访问的提示出现

遇到了同样的问题，必须要开全局代理才可以使用

**Comment by pan93412** [comment]
不太清楚，但我揣測是 shadowrockets 的設定問題。

**Reply by zq0x7a0x71** [reply]
同样的 shadowrockets 设置，在 nondanee 版本下可用，在本 repo 的版本下不可用。

设置方式：https://github.com/nondanee/UnblockNeteaseMusic/issues/779#issuecomment-917612470

**Reply by pan93412** [reply]
> 同样的 shadowrockets 设置，在 nondanee 版本下可用，在本 repo 的版本下不可用。
> 
> 设置方式：[nondanee#779 (comment)](https://github.com/nondanee/UnblockNeteaseMusic/issues/779#issuecomment-917612470)

相關邏輯原則上應該沒有改變。我沒用 shadowrockets 不好測試，歡迎有志之士協助抓蟲。

**Comment by pan93412** [comment, marked as answer]
#242 可能修正本問題。

**Reply by zq0x7a0x71** [reply]
确认已修复。谢谢  : )

**Reply by heeroyuyj** [reply]
> 确认已修复。谢谢 : )

下载最新 beta 版吗?

**Reply by zq0x7a0x71** [reply]
我是直接下载的源码，release 的新版本应该也修复了这个问题。

**Reply by heeroyuyj** [reply]
> 我是直接下载的源码，release 的新版本应该也修复了这个问题。

呃 好吧 因为我不会编译 我看正式版还是没变动, beta 版倒是很多个

**Reply by pan93412** [reply]
直接用 beta 版。release 不出只是因為我懶著打 Changelog。

**Reply by heeroyuyj** [reply]
> 直接用 beta 版。release 不出只是因為我懶著打 Changelog。

ok, 收到
